llvm/lib/Target/DirectX/DXILRootSignature.cpp

```diff
@@ -205,14 +304,35 @@ PreservedAnalyses RootSignatureAnalysisPrinter::run(Module &M,
 
     // start root signature header
     Space++;
-    OS << indent(Space) << "Flags: " << format_hex(RS.Flags, 8) << ":\n";
-    OS << indent(Space) << "Version: " << RS.Version << ":\n";
-    OS << indent(Space) << "NumParameters: " << RS.Parameters.size() << ":\n";
-    OS << indent(Space) << "RootParametersOffset: " << RSHSize << ":\n";
-    OS << indent(Space) << "NumStaticSamplers: " << 0 << ":\n";
+    OS << indent(Space) << "Flags: " << format_hex(RS.Flags, 8) << "\n";
+    OS << indent(Space) << "Version: " << RS.Version << "\n";
+    OS << indent(Space) << "NumParameters: " << RS.Parameters.size() << "\n";
+    OS << indent(Space) << "RootParametersOffset: " << RSHSize << "\n";
```

Comment:
Technically a pre-existing bug, but this needs to print `RS.RootParametersOffset`, not `RSHSize`